mlxtend/frequent_patterns/tests/test_fpbase.py

```diff
@@ -133,6 +147,20 @@ def test_max_len(self):
         max_len = np.max(res_df2['itemsets'].apply(len))
         assert max_len == 2
 
+    def test_low_memory_flag(self):
+        try:
+            with captured_output() as (out, err):
+                _ = self.fpalgo(self.df, low_memory=True, verbose=1)
+                print(out.getvalue())
+        except TypeError:
+            # If there is no low_memory argument, don't run the test.
+            assert True
```

Comment:
Hm, since it's a function, that's a tricky one ... one solution, which is not that much better, would be

```
import inspect

    def test_low_memory_flag(self):

        if 'low_memory' in inspect.getargspec(self.fpalgo)[0]:
             with captured_output() as (out, err):
                 _ = self.fpalgo(self.df, low_memory=True, verbose=1)
                 print(out.getvalue())
         else:
             # If there is no low_memory argument, don't run the test.
             assert True
```

Reply:
Actually, I like that better. I've changed this test to use `inspect.signature` and removed that pesky debug `print` that snuck through

Reply:
sounds good!